.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,78 @@
+name: "Docker Build"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
One last thing, can you add `dev/0.15.0` branch to these arrays (**not** push? That way we can test it live (don't worry about the `tags: | \n tycrek/ass:master` below, that can stay as-is).